x/metadata/keeper/objectstorelocator.go

```diff
@@ -0,0 +1,220 @@
+package keeper
```

Comment:
the name of this file should be consistent with the types and the proto file names.

Reply:
good point..changed

Reply:
changed